UPGRADE.md

```diff
@@ -12,6 +12,10 @@ High notes are now taken into account in vertical spacings. If you would like to
 
 `\grelooseness` is now `-1` by default. This should prevent orphaned syllable at end of score. If you prefer the old behavior, use `\def\grelooseness{\looseness}` befor including your scores.
 
+### Stemmed oriscus flexus orientation
```

Comment:
This is a problem in 4.1.1 itself.  I'll fix it for develop since it's too late for 4.1.1.